### PR TITLE
#1243 Support for custom gce images

### PR DESCRIPTION
**Pull request by aperepel**

### CODE REVIEW

**Comment by nathanleclaire** [comment]
cc @jeffmendoza PTAL

File: drivers/google/compute_util.go [hunk 1]
```diff
@@ -29,8 +29,9 @@ type ComputeUtil struct {
 }
 
 const (
-	apiURL             = "https://www.googleapis.com/compute/v1/projects/"
-	imageName          = "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1404-trusty-v20150316"
+	apiURL           = "https://www.googleapis.com/compute/v1/projects/"
+	defaultImageName = "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1404-trusty-v20150316"
+	//	imageName          = "https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/centos-7-v20150603"
```
Comment (jeffmendoza) [inline drivers/google/compute_util.go]: remove comment.

File: docs/index.md [hunk 2]
```diff
@@ -1225,7 +1226,10 @@ Options:
  - `--google-disk-size`: The disk size of instance.
  - `--google-disk-type`: The disk type of instance.
 
-The GCE driver will use the `ubuntu-1404-trusty-v20150316` instance type unless otherwise specified.
+The GCE driver will use the Ubuntu-based instance image unless otherwise specified. To obtain a
+list of image URLs run:
+```gcloud compute images list --uri
```
Comment (jeffmendoza) [inline docs/index.md]: this is not rendering right, move gcloud down one line after the quotes.

**Comment by jeffmendoza** [comment]
I was able to create a 15.04 vm, but machine ran into errors bootstrapping. The VM was created correctly with the specified image, so this code is working.

**Comment by aperepel** [comment]
FYI, the `centos-7-v20150603` image worked without issues for me.

**Comment by aperepel** [comment]
Hi, checking on the status of this PR, over a month old now (and now reports a merge conflict). Is there anything blocking the PR other than the conflict? I can resolve it again, but if we wait for another month, there will be another one potentially, please advise.

**Comment by nathanleclaire** [comment]
@aperepel Just dropped the ball due to being busy with DockerCon and the release.  Everything looks pretty good to me so if you rebase I think we can go ahead and move it upstream.  cc @ehazlett

**Comment by dgageot** [comment]
Hi @aperepel are you ok to rebase your PR?

**Comment by dgageot** [comment]
This will have to wait for https://github.com/docker/machine/pull/2087 to be merged

**Comment by dgageot** [comment]
I created a rebased version of this PR https://github.com/docker/machine/pull/2113

**Pull request closed without merging**